gradio.egg-info/requires.txt

```diff
@@ -16,3 +16,5 @@ pydub
 requests
 uvicorn
 Jinja2
+httpx
```

Comment:
If we are going to go with `httpx`, can we replace our usage of `aiohttp` with `httpx` entirely and remove `aiohttp` as a dependency?

Reply:
Yeah planning to it